Susi/Helpers/ControllerConstants.swift

```diff
@@ -273,5 +273,12 @@ class ControllerConstants {
         static let reportIssueURL = "https://github.com/fossasia/susi_iOS/issues"
         static let licenseURL = "https://github.com/fossasia/susi_iOS/blob/master/LICENSE"
     }
+    
+    struct Logout {
+        static let title = "Logout"
+        static let message = "Are you sure , you want to logout?"
```

Comment:
It is unprofessional to keep space before a comma. so change it to `Are you sure, you want to log out?`

Reply:
changes done , please review